scripts/csvcut.py

```diff
@@ -0,0 +1,65 @@
+#!/home/david/miniconda3/bin/python3
+from argparse import ArgumentParser
+from signal import signal, SIGPIPE, SIG_DFL
+import numpy as np
+import sys
+from utils import *
+
+DESCRIPTION = 'csvcut - Select some columns from csv streem. Could change order of fields.'
+EXAMPLES = 'example: csvcut -f 1,2 stat.txt'
+
+
+def print_row(row, output_stream):
+    """
+    Prints a row in csv format
+    :param row: row represented as a list of columns
+    :param output_stream: a stream to pretty print the row
+    """
+    output_line = ''
+    for index, column in enumerate(row):
```

Comment:
Please read about join method of python string. You can achieve the same with ','.join(...)